data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -369,50 +368,162 @@
     f"for low life expectancy and is low income?"
 )
 
+# Score M Low Income Change
+## TODO Note that this changes the syntax of some columns to have fewer ands -- check output
+## A small point -- this file is long and unwieldy and it might make sense to streamline at some point?
```

Comment:
Agreed, let's think about that as part of the Big Refactor of Fields, Models, etc.